### PR TITLE
ARR Installation and HTTP Redirection MOD

### PR DESCRIPTION
**Pull request by stajkowski**
Installs ARR and enabled the HTTP Redirection Feature.

### CODE REVIEW

**Comment by btm** [comment]
@stajkowski two of your commits were made with the address of ```Author: Brian Stajkowski <stajkowski@stajkowski-ubu.(none)>``` which obviously doesn't match the email address on your Github account so Curry (the bot) can't match them, that's all. I've manually marked this pull request as CLA approved.

**Comment by EasyAsABC123** [comment]
:+1:

File: recipes/mod_http_redirection.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#
+# Author:: Seth Chisamore (<schisamo@opscode.com>)
```
Comment (ksubrama) [inline recipes/mod_http_redirection.rb]: Take Credit :)

File: recipes/install_arr.rb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+#Start IIS WAS
+windows_batch "Start IIS WAS" do
+  code <<-EOH
+  net start was /y
+  EOH
+  action :run
+end
+
+#Start IIS W3SVC
+windows_batch "Start IIS" do
```
Comment (ksubrama) [inline recipes/install_arr.rb]: These should now use the service resource to start and stop.

File: recipes/install_arr.rb [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  arrRewrite = "http://download.microsoft.com/download/6/7/D/67D80164-7DD0-48AF-86E3-DE7A182D6815/rewrite_amd64_en-US.msi"
+else
+  externalCache = "http://download.microsoft.com/download/4/D/F/4DFDA851-515F-474E-BA7A-5802B3C95101/ExternalDiskCache_x86_en-US.msi"
+  webFarm = "http://download.microsoft.com/download/5/7/0/57065640-4665-4980-A2F1-4D5940B577B0/webfarm_v1.1_amd64_en_US.msi"
+  requestRouter = "http://download.microsoft.com/download/6/3/D/63D67918-483E-4507-939D-7F8C077F889E/requestRouter_x86.msi"
+  arrRewrite = "http://download.microsoft.com/download/6/9/C/69C1195A-123E-4BE8-8EDF-371CDCA4EC6C/rewrite_x86_en-US.msi"
+end
+
+#Stop IIS WAS
+windows_batch "Stop IIS" do
```
Comment (ksubrama) [inline recipes/install_arr.rb]: Use service resource to stop IIS.

File: recipes/install_arr.rb [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  net stop was /y
+  EOH
+  action :run
+end
+
+################################
+### Install ARR ################
+################################
+
+#Install x64 External Disk Cache
```
Comment (ksubrama) [inline recipes/install_arr.rb]: Comment is stale - presumably it installs the appropriate 32 or 64bit package.

**Comment by ksubrama** [comment]
Hey!  I know this is a really old pull-request.  If you're still around and wish to clean it up, I'd be happy to merge this in.  If not, I'll get to it after some other refactoring work and merge what I'm able to.

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by EasyAsABC123** [comment]
closing this pull request due to the amount of time since this was created.  if someone would like to recreate this PR I completely understand

**Pull request closed without merging**